marconi-cardano-chain-index/marconi-cardano-chain-index.cabal

```diff
@@ -121,15 +121,15 @@ library
   build-depends:
     , aeson
     , async
-    , base                  >=4.9 && <5
+    , base                       >=4.9 && <5
     , bytestring
     , comonad
     , directory
     , filepath
     , lens
     , mtl
     , nonempty-containers
-    , optparse-applicative
+    , optparse-applicative-fork
```

Comment:
I don't find this package anywhere, where does it come from and why is it needed?

Reply:
It's used by `cardano-node`: https://github.com/CardanoSolutions/cardano-node/blob/release/8.1.2/cardano-node/cardano-node.cabal#L178C50-L178C50

I've added it here to allow me to combine the CLI parsers from `marconi` and `cardano-node` in `marconi-sidechain-node`.

It lives here: https://github.com/input-output-hk/optparse-applicative

Reply:
Hmm... I hate the idea of changing the library because another one is used downstream but I guess we don't have the choice here.